### PR TITLE
Deliver MicroProfile 2.2 grouping feature

### PR DESCRIPTION
**Pull request by NottyCode**
Deliver the MicroProfile 2.2 grouping feature so it can be released in the next release.

### CODE REVIEW

**Comment by NottyCode** [comment]
#build

**Comment by NottyCode** [comment]
#build this please Miss Libby

File: dev/com.ibm.websphere.appserver.features/visibility/public/microProfile-2.2/resources/l10n/com.ibm.websphere.appserver.microProfile-2.2.properties [hunk 1]
```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2018 IBM Corporation and others.
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/public/microProfile-2.2/resources/l10n/com.ibm.websphere.appserver.microProfile-2.2.properties]: Wrong date

File: dev/com.ibm.websphere.appserver.features/visibility/public/microProfile-2.2/resources/l10n/com.ibm.websphere.appserver.microProfile-2.2_cs.properties [hunk 2]
```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2016 IBM Corporation and others.
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/public/microProfile-2.2/resources/l10n/com.ibm.websphere.appserver.microProfile-2.2_cs.properties]: Defintiely wrong date

**Comment by fmhwong** [comment]
@NottyCode OpenTracing has a new auto feature which needs to be changed to GA too.  https://github.com/OpenLiberty/open-liberty/blob/master/dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.mpOpenTracing1.3-mpRestClient1.2.feature

File: dev/com.ibm.websphere.appserver.features/visibility/public/opentracing-1.3/com.ibm.websphere.appserver.opentracing-1.3.feature [hunk 3]
```diff
@@ -19,5 +19,5 @@ IBM-API-Package: io.opentracing;  type="third-party",\
          com.ibm.websphere.appserver.thirdparty.opentracing.0.31.0; location:="dev/api/third-party/,lib/"; mavenCoordinates="io.opentracing:opentracing-api:0.31.0"
 -jars=com.ibm.websphere.appserver.spi.opentracing.1.3; location:=dev/spi/ibm/
 -files= dev/spi/ibm/javadoc/com.ibm.websphere.appserver.spi.opentracing.1.3_1.0-javadoc.zip
-kind=noship
+kind=ga
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/public/opentracing-1.3/com.ibm.websphere.appserver.opentracing-1.3.feature]: opentracing-1.3 isn't part of MP 2.2

Comment (fmhwong) [reply dev/com.ibm.websphere.appserver.features/visibility/public/opentracing-1.3/com.ibm.websphere.appserver.opentracing-1.3.feature]: mpOpenTracing-1.3 is depending on opentracing-1.3.

**Comment by bsbyrd1** [comment]
Doesn't look like ID Review is needed for these message files. If you require review, please let us know. Thanks.

File: dev/com.ibm.websphere.appserver.features/visibility/public/microProfile-2.2/resources/l10n/com.ibm.websphere.appserver.microProfile-2.2_cs.properties (hunk 2 above)
Comment (NottyCode) [reply dev/com.ibm.websphere.appserver.features/visibility/public/microProfile-2.2/resources/l10n/com.ibm.websphere.appserver.microProfile-2.2_cs.properties]: The information in this file is substantively the same as this file:

dev/com.ibm.websphere.appserver.features/visibility/public/microProfile-2.2/resources/l10n/com.ibm.websphere.appserver.microProfile-2.2_cs.properties

which has the 2016 date. Given there is only 1 character difference I don't think that constitutes a substantial enough change for a different copyright year.

File: dev/com.ibm.websphere.appserver.features/visibility/public/microProfile-2.2/resources/l10n/com.ibm.websphere.appserver.microProfile-2.2.properties (hunk 1 above)
Comment (NottyCode) [reply dev/com.ibm.websphere.appserver.features/visibility/public/microProfile-2.2/resources/l10n/com.ibm.websphere.appserver.microProfile-2.2.properties]: The information in this file is substantively the same as this file:

dev/com.ibm.websphere.appserver.features/visibility/public/microProfile-2.2/resources/l10n/com.ibm.websphere.appserver.microProfile-2.2.properties

which has the 2018 date. Given there is only 1 character difference I don't think that constitutes a substantial enough change for a different copyright year.

**Comment by NottyCode** [comment]
#build this please Libby

**Comment by skasund** [comment]
Message review complete from L2

**Pull request merged**